src/renate/benchmark/datasets/wild_time_data.py

```diff
@@ -90,6 +90,7 @@ def setup(self) -> None:
             "time_step": available_time_steps(self._dataset_name)[self.data_id],
             "data_dir": self._data_path,
             "in_memory": self._dataset_name != "fmow",
+            "transform": None if self._dataset_name != "fmow" else lambda x: x,
```

Comment:
Can you try this on multigpu? I reckon this DDP won't like this lambda.

Reply:
you mean, the function needs to be pickable?

Reply:
yes

Reply:
I've just noticed that 3 lines below is yet another lambda function...

Reply:
I've tested it, it doesn't cause any problems.